### PR TITLE
FHAC-977: The "java:comp/env/" prefix has been removed from the hibernate datasources.

### PR DESCRIPTION
**Pull request by danielrf123**
1. The "java:comp/env/" prefix has been removed from the hibernate datasources.  It was causing errors during the deployments to Weblogic and WAS. 
2. Excluded the javax.servlet-api-3.0.1.jar lib from WAS ear packaging in was/pom.xml.

**NOTE**: This commit had major issues with merging from CONNECT.  Looks like I overwrote the changes from Minh's PR #1287 that was merged today.  Will discuss this with @mhpnguyen tomorrow.

Files that got committed that shouldn't have from the merge with 1287:
- DocQueryRetrieveSelfTest-soapui-project.xml
- DocQuerytestsforUDDI-soapui-project.xml
- EntityXDRTests-soapui-project.xml
- PatientCorrelation-soapui-project.xml
- Standard/pom.xml

### CODE REVIEW

**Comment by danielrf123** [comment]
New commits have been made that would add back the file updates for FHAC-1002 (PR 1287)

File: Product/Production/Deploy/was/pom.xml [hunk 1]
```diff
@@ -105,6 +105,7 @@
                         lib/geronimo-servlet_3.0_spec-1.0.jar,
                         lib/geronimo-javamail_1.4_spec-1.7.1.jar,
                         lib/servlet-api-2.5.jar,
+						lib/javax.servlet-api-3.0.1.jar,
```
Comment (mhpnguyen) [inline Product/Production/Deploy/was/pom.xml:108]: How about other servers (weblogic/wildfly/jboss)?

Comment (danielrf123) [inline Product/Production/Deploy/was/pom.xml:108]: Good catch.  Only weblogic is the other one that needs to exlude it.

**Comment by sailajaa** [comment]
I would like to know why soapui projects are included in this PR. I understand its spaces, but I believe   this PR should not include any of those space changes.

**Comment by TabassumJafri** [comment]
Reviewed PR. Above comments need to be addressed before merging.

**Comment by mhpnguyen** [comment]
endorse for merging without comments.

**Pull request merged**